### PR TITLE
PR for: setting up linters, setting-up-your-repository and checking-repository-health

### PR DESCRIPTION
**Pull request by jomadoye**
#### What does this PR do?
1. This PR contains outputs for "setting up linters"
2. This PR contains outputs for "setting-up-your-repository"
3. This PR contains outputs for "checking-repository-health"

#### Description of Task to be completed?
1. Screenshots of ESLint Airbnb style violations in the terminal and HoundCI comment on a PR
2. The link to the repository for your proposed project and at least one `Pull Request`
3. Screenshot of your code coverage and GPA badges in your repo's README file

#### How should this be manually tested?
1. By checking the outputs from my repository (setting up linters) , it contains the screenshots in the README file
2. By checking the outputs from my repository (setting-up-your-repository) , it contains the link in the README file
3. By checking the outputs from my repository (checking-repository-health) , it contains the screenshots in the README file

#### Any background context you want to provide?
N/A

### CODE REVIEW

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- **Deliver**: two screenshots of ESLint [Airbnb](https://github.com/airbnb/javascript)  style violations in the terminal and [HoundCI](https://houndci.com/) comment on a PR
+- **With**: *independently*
+
+#### Link to tutput
+- [Setting Up Linters Output](https://github.com/andela-jomadoye/Inverted-index-repo/pull/1)
+
+#### Link to Images
+- [Eslint image](https://www.dropbox.com/s/rbanrlwqtlmfw84/Screen%20Shot%202017-03-17%20at%204.48.49%20PM.png?dl=0)
+
+- [HoundCI image](https://www.dropbox.com/s/5h4ye313khknrwv/Screen%20Shot%202017-03-17%20at%205.57.42%20PM.png?dl=0)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md:12]: Great Job!

File: 06-building-a-program-start-to-finish/checking-repository-health/output/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: `Code Climate` and `Coveralls`
+- **Deliver**: a screenshot of your code coverage and GPA badges in your repo's README file
+- **With**: *independently*
+
+#### link to screenshot of code climate and coveralls badges
+- [badges](https://www.dropbox.com/s/a8n7dd5babwsdwj/Screen%20Shot%202017-03-18%20at%203.28.00%20PM.png?dl=0)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/checking-repository-health/output/README.md:7]: Cool!

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+#### Link to project repository
+- [Inverted-Index Repo](https://github.com/andela-jomadoye/Inverted-index-repo)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:7]: You should improve your ReadMe to accurately describe the project.

[This](https://gist.github.com/q-ode/0b6029f7def28d3006f36d09ea6b849f) should guide you. Please share with your class.

**Review by q-ode — COMMENTED** [review]
Good start!

Please check the comments.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md (hunk 3 above)
Comment (jomadoye) [reply 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:7]: ok, thanks.

**Review by q-ode — COMMENTED** [review]
Great Job!

**Pull request merged**